### PR TITLE
feat: added connected clusters to the ocm view

### PR DESCRIPTION
**Pull request by TomerFi**
- docs: typo
- feat: added view for local connected k8s clusters
- chore: finalize tree-view for ocm connected clusters
- chore: update views naming convention

### CODE REVIEW

File: integration-tests/test-files/cmd.createEnvironment.test.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ async function sleep(ms: number): Promise<void> {
 // Test cases for the the ocm-vscode-extension.createLocalEnvironment command
 suite('Create local environment command', () => {
 	const fakeBuildSuccessMsg = 'this is a fake message';
-	const buildEnvironmentDelayMS = 500;
+	const buildEnvironmentDelayMS = 1000;
```
Comment (ilan-pinto) [inline integration-tests/test-files/cmd.createEnvironment.test.ts:19]: :)

File: src/providers/connectedClusters.ts [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+	readonly onDidChangeTreeData: vscode.Event<void> = this.configRefreshedEmitter.event;
+
+	constructor() {
+		this.updateK8SConfig();
+	}
+
+	refresh(): void {
+		this.updateK8SConfig();
+		this.configRefreshedEmitter.fire();
+	}
```
Comment (ilan-pinto) [inline src/providers/connectedClusters.ts:58]: fire?????

File: src/providers/connectedClusters.ts [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+	private async getNamespacedResourceLists(
+		spec: k8s.V1CustomResourceDefinitionSpec,
+		k8sCustomObjApi: k8s.CustomObjectsApi,
+		listResourcesPromises: Promise<void>[],
+		customResources: OcmResource[]): Promise<void> {
+
+		let k8sCoreApi = this.kubeConfig.makeApiClient(k8s.CoreV1Api);
+		let namespacesApiResponse = await k8sCoreApi.listNamespace();
+		if (namespacesApiResponse.response.statusCode === 200) {
```
Comment (ilan-pinto) [inline src/providers/connectedClusters.ts:121]: what about error handling?

File: src/providers/connectedClusters.ts [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+		spec: k8s.V1CustomResourceDefinitionSpec,
+		k8sCustomObjApi: k8s.CustomObjectsApi,
+		listResourcesPromises: Promise<void>[],
+		customResources: OcmResource[]): Promise<void> {
+
+		spec.versions.forEach(async ver => {
+			listResourcesPromises.push(
+				k8sCustomObjApi.listClusterCustomObject(spec.group, ver.name, spec.names.plural)
+				.then(crApiResponse => {
+					if (crApiResponse.response.statusCode === 200) {
```
Comment (ilan-pinto) [inline src/providers/connectedClusters.ts:151]: error handling

**Review by ilan-pinto — APPROVED** [review]
dor demo it looks good.

File: integration-tests/test-files/cmd.createEnvironment.test.ts (hunk 1 above)
Comment (TomerFi) [reply integration-tests/test-files/cmd.createEnvironment.test.ts:19]: GH runners are much slower than my station...

https://github.com/ilan-pinto/ocm-vscode-extension/actions/runs/2350333292

File: src/providers/connectedClusters.ts (hunk 2 above)
Comment (TomerFi) [reply src/providers/connectedClusters.ts:58]: this fires an event informing vscode on the data being refreshed.

https://code.visualstudio.com/api/references/vscode-api#EventEmitter%3CT%3E

File: src/providers/connectedClusters.ts (hunk 3 above)
Comment (TomerFi) [reply src/providers/connectedClusters.ts:121]: not for the demo, nither does testing.
for now, for error in k8s, an empty list is returned to not f-up the demo.

File: src/providers/connectedClusters.ts (hunk 4 above)
Comment (TomerFi) [reply src/providers/connectedClusters.ts:151]: not for the demo, nither does testing.
for now, for error in k8s, an empty list is returned to not f-up the demo.

**Pull request merged**